test/admin-hpa/Rakefile

```diff
@@ -6,35 +6,75 @@ require 'capybara'
 require 'colorize'
 require_relative 'features/support/browserstack_settings'
 
+
+GROUP_SIZE = ENV['GROUP_SIZE'] || 1
+NODES = ENV['NODES'] || 1
+
 File.truncate('rerun.txt', 0) if File.exist?('rerun.txt')
 File.truncate('rerun0.txt', 0) if File.exist?('rerun0.txt')
 
+def run_in_groups(group_size = 1)
+  driver = Rake.application.top_level_tasks.first.split(':').last.gsub('parallel_', 'bs_')
+  @exit_code_array = []
+  features = Dir["features/*"]
+  array_of_features = features.each_slice(group_size.to_i).to_a
+  group_index = 0
+  features_to_run = Queue.new
+  nodes = NODES.to_i
+  array_of_features.each {|s| features_to_run.push s}
+  parent_rerun_file = "reports/parent_rerun_#{driver}.txt"
+  File.open(parent_rerun_file, 'w')
+  File.delete(parent_rerun_file) if File.exists?(parent_rerun_file)
```

Comment:
Shouldn't this line be before you open the file for writing?

Reply:
updated, silly mistake 👍